worker/sshserver/manifold.go

```diff
@@ -88,13 +98,18 @@ func (config ManifoldConfig) startWrapperWorker(context dependency.Context) (wor
 	}
 
 	w, err := config.NewServerWrapperWorker(ServerWrapperWorkerConfig{
-		NewServerWorker: config.NewServerWorker,
-		Logger:          config.Logger,
-		FacadeClient:    client,
+		NewServerWorker:      config.NewServerWorker,
+		Logger:               config.Logger,
+		FacadeClient:         client,
+		NewSSHServerListener: config.NewSSHServerListener,
```

Comment:
I don't think we need this on the manifold or worker config, as it's test only, and the only tests that will execute it are in server.go we can simplify this by removing it from here and worker.go and leave it as an option only on server.go & allow it to be nillable I think? Then in server.go check if it's nil and create a listener?

Reply:
This is the common pattern, we'll just stick to it for consistency.